src/slimevr101.md

```diff
@@ -2,29 +2,72 @@
 
 ## What is SlimeVR?
 
-SlimeVR is a low-cost solution to full-Body tracking in virtual reality. It uses forward kinematics to build a model of your skeleton calculated from the rotation of each individual tracker, with your headset and controllers being the only absolute known position.
+SlimeVR is an affordable solution for full-body tracking in virtual reality. It uses forward kinematics[^note] to create a model of your body by calculating the position of each tracker based on its rotation. The only fixed point is your headset and controllers, which are used as reference positions.
+
+Since the headset is the only fixed data point, SlimeVR doesn’t need extra tracking equipment like lighthouses. It relies on Inertial Measurement Units (IMUs) to track the rotation of each device. The more IMUs used, the more tracking points are available for your body.
+
+[^note]: Forward kinematics is the process of calculating the position of a body part (like a foot or arm) based on the angles of joints. Given how your joints (like knee or elbow) are positioned, forward kinematics tells you where your foot or arm will be in space. It's like figuring out where your foot will go when you bend your leg a certain way.
```

Comment:
I'd rather say "angles of the bones", as joints can't have angles, since they are point for the most part.